frontend/src/components/feed/Feed.js

```diff
@@ -36,10 +36,19 @@ const Feed = ({ navigate }) => {
             </button>
           <div id='feed' role="feed">
               {posts.map(
-                (post) => ( <Post post={ post } key={ post._id } /> )
+                (post) => ( <a href= {`#${post._id}`} key={post._id}><Post post={ post } key={ post._id } /></a> )
```

Comment:
can you please explain what is key=post_id doing?

Reply:
https://kentcdodds.com/blog/understanding-reacts-key-prop

gives a number to the list component so react knows which link is which i think...